src/__tests__/index.test.js

```diff
@@ -50,6 +50,37 @@ describe('normalize', () => {
     expect(normalize(input, article)).toMatchSnapshot();
   });
 
+  it('normalizes recursive payloads', () => {
+    const user = new schema.Entity('users', {
+      comments: [comment],
```

Comment:
@paularmstrong I agree that in this case this is indeed the solution. However in my project I have `schemas/user.js` and `schemas/comment.js`.

How can I then solve the following:
```
// user.js
import { schema } from 'normalizr';
import { comment } from './comment.js';

export const user = new schema.Entity('users', {
  comments: [comment],
});
```
and 
```
// comment.js
import { schema } from 'normalizr';
import { user } from './user.js';

export const comment = new schema.Entity('comments', {
  user: user,
});
```